app/helpers/session_helpers.py

```diff
@@ -21,13 +22,13 @@ def wrapped_function(*args, **kwargs):
     return wrapped_function
 
 
-def with_session_store(function):
+def with_session_store(function: Callable) -> Any:
     """Adds the `session_store` as an argument.
     Use on flask request handlers or methods called by flask request handlers.
     """
 
     @wraps(function)
-    def wrapped_function(*args, **kwargs):
+    def wrapped_function(*args: Any, **kwargs: Any) -> Any:
```

Comment:
its looks identical as above why is it `Any`

Reply:
Since ** unpacks dictionaries and kwargs are keyword arguments shouldn't it have type `dict` or `Mapping` instead of `Any`?

Reply:
The above existing one was wrong, I updated it to Any. For args and kwargs, they are tuple are dicts, but that is not how it is supposed to be typed. It should be typed with the actual types they can actually hold. https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

It is at the moment, any as this is a generic decorator that can be called by any function, and we can't infer the type of the callers args in 3.9. As per the PR description:
> Note: Decorator type hints return Any due to a lack of support by the language. It has been improved in Py 3.10, therefore, we can properly type check decorators once we are on Py 3.10 using [ParamSpec](https://peps.python.org/pep-0612/).